services/src/northbound/src/main/java/org/openkilda/northbound/service/BasicService.java

```diff
@@ -81,4 +88,43 @@ default InfoData validateInfoMessage(final Message requestMessage,
         }
         return data;
     }
+
+    /**
+     * Validates switch existing.
+     *
+     * @param correlationId the correlation id.
+     * @param data list with source and destination switches.
+     * @throws org.openkilda.messaging.error.MessageException if switch not found.
+     */
+    @SuppressWarnings("squid:CallToDeprecatedMethod")
+    default void validateSwitches(String correlationId, List<SwitchInfoData> data, FlowPayload flowPayload) {
```

Comment:
What is the reason to expose validateSwitches which relies on FlowPayload as a method of all NB services? 

BTW, the existing BasicService.validateInfoMessage provides functionality which has no reason to be public.